src/Symfony/Component/Scheduler/Tests/Messenger/SchedulerTransportTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Tests\Messenger;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Message\RedispatchMessage;
+use Symfony\Component\Scheduler\Exception\LogicException;
+use Symfony\Component\Scheduler\Generator\MessageGeneratorInterface;
+use Symfony\Component\Scheduler\Messenger\ScheduledStamp;
+use Symfony\Component\Scheduler\Messenger\SchedulerTransport;
+
+class SchedulerTransportTest extends TestCase
+{
+    public function testGetFromIterator()
+    {
+        $messages = [
+            (object) ['id' => 'first'],
+            (object) ['id' => 'second'],
+        ];
+        $generator = $this->createConfiguredMock(MessageGeneratorInterface::class, [
+            'getMessages' => $messages,
+        ]);
+        $transport = new SchedulerTransport($generator);
+
+        foreach ($transport->get() as $envelope) {
+            $this->assertInstanceOf(Envelope::class, $envelope);
+            $this->assertNotNull($envelope->last(ScheduledStamp::class));
+            $this->assertSame(array_shift($messages), $envelope->getMessage());
+        }
+
+        $this->assertEmpty($messages);
+    }
+
+    public function testAckIgnored()
+    {
+        $transport = new SchedulerTransport($this->createMock(MessageGeneratorInterface::class));
+
+        $transport->ack(new Envelope(new \stdClass()));
+
+        $this->assertTrue(true); // count coverage
```

Comment:
should be replace by expectNotToPerformAssertions at the start of the method

Reply:
The `expectNotToPerformAssertions` works different: it's doesn't count code coverage. **This** test covers the logic, so should be counted in coverage.

Reply:
I've recheked. Useless test.